modules/common/src/test/java/com/ibm/cloud/cloudant/internal/ErrorTransformInterceptorTest.java

```diff
@@ -50,6 +50,7 @@ public class ErrorTransformInterceptorTest {
     private static final String MOCK_REASON = "some test reason";
     private static final String MOCK_REASON_K = "reason";
     private static final String MOCK_REQ_ID = "338da230c5";
+    private static final String MOCK_REQ_ID2 = "455fe7724";
```

Comment:
REQ_ID2 is kind of low on recognizability level, I'd rather have MOCK_PREFERRED_REQ_ID, similar to python

Reply:
I used `MOCK_COUCH_REQ_ID` and `MOCK_REQ_ID` in cff6dfad to better reflect which value is in which header.